events/types.go

```diff
@@ -18,6 +18,9 @@ const (
 
 	// AppPurchasedEventType indicates which type the struct is. Could make it easier for us data map this event from the database
 	AppPurchasedEventType EventType = "app.purchase.created"
+
+	// AppBundleSubscriptionMigratedEventType defines the event related to when a subscription is migrate to use the bundle
+	AppBundleSubscriptionMigratedEventType EventType = "app.subscription.bundle.migrated"
```

Comment:
```suggestion
	AppBundleSubscriptionMigratedEventType EventType = "app.subscription.migrated"
```
Let's go with this as the name of the event. Also, if you can move this up underneath the group of events related to subscriptions that would be great.

I am thinking something like: `item.relatedItem.action` and if there's no related item it would be `item.action`

So this would be `app.x.created` and if there's no related item then it would be `app.created` or `app.updated` or `publisher.updated`. Then if we want more information about the event, it should be inside the event's data and detail.

For this event, it would be `app.subscription.migrated` and the detail data would be `MigratedToBundle` and if the subscription got migrated to a free version the `MigratedToFree` could be a flag.

Thoughts?

Reply:
I really liked the proposed pattern. I also liked the suggestion to change the event name for `app.subscription.migrated` based on this rationale. It's also easier to guess the event subject. 

For instance, if we are now creating an event related to a bundle, we can do something like that `bundle.app.removed`?

Reply:
Yes sir, precisely. That tells me an app was removed from a bundle, which makes perfect sense.

Reply:
Awesome! 🙌 changes submitted to the branch ✅